.github/workflows/release.yml

```diff
@@ -0,0 +1,25 @@
+name: Upload Python Package
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install setuptools wheel twine
+    - name: Build and publish
+      env:
+        TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+        TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+      run: |
+        make release
```

Comment:
This looks good but isn't it going to fail if the package version was already uploaded by Travis?

Reply:
As this action is only supposed to run when a new tag( release) is created. I'm hoping this won't conflict with existing versions on PyPi.

Reply:
Travis also uploads packages when a tag is created, so if it uploads before the GitHub Action, then `twine` may return with an error.

In any case, I'm happy to merge this and try it if you want.  The GitHub Action will most likely be executed first since it's faster than Travis CI.  We could also disable the Travis CI deploy job.